src/base.js

```diff
@@ -95,10 +95,13 @@ export function VRDisplay(config) {
   // "Private" members.
   this.waitingForPresent_ = false;
   this.layer_ = null;
-  // In WebVR 2.0, the context's canvas doesn't have
-  // to be in the DOM at all; if we have to inject it,
-  // set orphanedLayer to true so we can remove it later
-  this.orphanedLayer = null;
+  // Keep track of the original parent of the source passed into
+  // `requestPresent`. While the fullscreenWrapper will be a child of the parent
+  // in most cases, we must keep track when there's no parent (like
+  // when it was never in the DOM, i.e. WebXR polyfill), or if the `source`
```

Comment:
"i.e." means "in other words" - I think you wanted "e.g." here (_exempli gratia_, "for example")